api/src/test/java/org/openmrs/module/openconceptlab/importer/SaverTest.java

```diff
@@ -1221,6 +1274,31 @@ public void importMapping_shouldRemoveConceptMapping() throws Exception {
 		assertThat(BooleanUtils.isTrue(term.getRetired()), is(false));
 	}
 
+	@Test

```

Comment:
This test passes even without this change, I just wrote it to verify that a similar problem wasn't happening with concept mappings.